src/sidebar/services/annotations-exporter.ts

```diff
@@ -108,8 +99,90 @@ export class AnnotationsExporter {
       Total users: ${uniqueUsers.length}
       Users: ${uniqueUsers.join(', ')}
       Total annotations: ${annotations.length}
-      Total replies: ${annotations.filter(isReply).length}
+      Total replies: ${replies.length}
       
       ${annotationsText}`;
   }
+
+  buildCSVExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      defaultAuthority = '',
+      displayNamesEnabled = false,
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, uniqueUsers, replies, extractUsername } = this._exportCommon(
+      annotations,
+      {
+        displayNamesEnabled,
+        defaultAuthority,
+      },
+    );
+
+    const escapeCSVValue = (value: string): string => {
+      // If the value contains a comma, newline or double quote, then wrap it in
+      // double quotes and escape any existing double quotes.
+      if (/[",\n]/.test(value)) {
+        return `"${value.replace(/"/g, '""')}"`;
+      }
+      return value;
+    };
```

Comment:
Might be a good idea to move this into a utility module that can be tested separately.